src/list.json

```diff
@@ -120,7 +120,18 @@
     ],
     "N": [],
     "O": [],
-    "P": [],
+    "P": [
+      {
+               "organization": "Percona",
+               "org_url": "https://www.percona.com/",
+               "tags": [ 
+                    "Percona swag"
+               ],
+               "description": "Submit at least four pull/merge requests (PRs) to hacktoberfest labelled. ",
```

Comment:
After you done with the changes please Re-Style it with 
`git checkout <your branch>`
`curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2332898/patch | git am`
`git push`

Reply:
let me fix this